### ISSUE TITLE
Game crashes when trying to draw useables on WON 1.1.0.7

### ISSUE THREAD

**Issue by hobokenn** [YaLTeR/BunnymodXT#94]

**Comment by YaLTeR** [comment]
Could you check this build https://github.com/YaLTeR/BunnymodXT/actions/runs/157730405, I might've fixed it.

**Comment by hobokenn** [comment]
still crashes

**Comment by hobokenn** [comment]
this crashes on ScreenToWorld here https://github.com/YaLTeR/BunnymodXT/blob/c62246711d1a23114b53e3e31e494c7a55252525/BunnymodXT/triangle_utils.cpp#L88 as far as i understand TriAPI can't access it in WON's hw.dll
![screentoworld](https://user-images.githubusercontent.com/50155487/94438875-59f08300-01a8-11eb-837d-ace3a508869e.png)

**Comment by YaLTeR** [comment]
Oh, huh, that's filled with garbage data. Need to look into the way the pointers are loaded by BXT to see if there's any way to tell when we don't have all of them.

**Comment by SmileyAG** [comment]
`ScreenToWorld` not exist in WON 1.1.0.7 (2001 year), some functions in **TriAPI** struct were added gradually with the new releases of engine builds.

The first declarated mention of `ScreenToWorld` begins with release of HLSDK 2.3 (2002 year): https://github.com/ScriptedSnark/hlsdk-versions/blob/main/hl-sdk-v2.3/Single-Player%20Source/common/triangleapi.h#L55

Looking to vtable at 1712 build I can confirm that it uses initial version of TriAPI struct: https://github.com/ScriptedSnark/hlsdk-versions/blob/main/hl-sdk-v2.1/SourceCode/common/triangleapi.h#L35

If you have thoughts that it may simply not be declared in the struct, but at the same time be in the engine, then I will upset - it's not (because I tried to find by looking at the order of the functions and using the specific byte sequence).

**Comment by SmileyAG** [comment]
TLDR; we should use some other alternative method instead of call `ScreenToWorld`, since it's not exist in that engine version.

**Comment by hobokenn** [comment]
what's the point of fixing this? do you need it in some video production?

**Comment by SmileyAG** [comment]
> what's the point of fixing this? do you need it in some video production?

Just showed a bit more explanation of issue, I don't think it worth to makes workaround for fixing it, since nobody running at that oldest WON engines.
